src/CompanyService.Models.Broker/Models/UserData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace LT.DigitalOffice.UserService.Models.Broker.Models
+{
```

Comment:
Is it works? I think in namespase of broker models must not be a service name

Reply:
Fixed